core/src/test/resources/test_tumor_type_summaries.txt

```diff
@@ -316,22 +316,22 @@ KIT	V560D	Gastrointestinal Stromal Tumor	KIT, a receptor tyrosine kinase, is rec
 KIT	P577_D579del	Gastrointestinal Stromal Tumor	KIT, a receptor tyrosine kinase, is recurrently mutated in gastrointestinal stromal tumors.	The KIT P577_D579del alteration is known to be oncogenic.	The multikinase KIT and PDGFRA inhibitors imatinib, sunitinib and regorafenib are FDA-approved for the treatment of patients with gastrointestinal stromal tumors (GIST) harboring KIT exon 11 alterations, such as the KIT P577_D579del mutation. The NCCN states that approximately 90% of patients with advanced GIST harboring KIT exon 11 mutations experience clinical benefit when treated with standard-dose 400 mg imatinib.
-KIT	D816V	Gastrointestinal Stromal Tumor	KIT, a receptor tyrosine kinase, is recurrently mutated in gastrointestinal stromal tumors.	The KIT D816V mutation is known to be oncogenic.	Select KIT hotspot exon 17 mutations have been identified in disease progression gastrointestinal stromal tumors (GIST) collected from patients treated with either multikinase inhibitor imatinib or sunitinib and for which the multikinase inhibitors regorafenib and sorafenib are FDA-approved and NCCN-compendium listed respectively. However, the KIT exon 17 D816V mutation is not a hotspot and has not been functionally or clinically validated, therefore the clinical utility of regorafenib and sorafenib in patients with KIT D816V mutant gastrointestinal stromal tumors is unknown.
-KIT	D816F	Gastrointestinal Stromal Tumor	KIT, a receptor tyrosine kinase, is recurrently mutated in gastrointestinal stromal tumors.	The KIT D816F mutation is known to be oncogenic.	Select KIT hotspot exon 17 mutations have been identified in disease progression gastrointestinal stromal tumors (GIST) collected from patients treated with either multikinase inhibitor imatinib or sunitinib and for which the multikinase inhibitors regorafenib and sorafenib are FDA-approved and NCCN-compendium listed respectively. However, the KIT exon 17 D816F mutation is not a hotspot and has not been functionally or clinically validated, therefore the clinical utility of regorafenib and sorafenib in patients with KIT D816F mutant gastrointestinal stromal tumors is unknown.
+KIT	D816V	Gastrointestinal Stromal Tumor	KIT, a receptor tyrosine kinase, is recurrently mutated in gastrointestinal stromal tumors.	The KIT D816V mutation is known to be oncogenic.	The exon 17 KIT D816V mutation has been identified in disease progression gastrointestinal stromal tumors (GIST) collected from patients treated with either multikinase inhibitors imatinib or sunitinib. While there is promising clinical data in patients with KIT D816-mutant advanced systemic mastocytosis treated with the multikinase inhibitor avapritinib, its clinical utility in patients with KIT D816V mutant gastrointestinal stromal tumors is unknown.
```

Comment:
curious - how does the system choose between the summary under D816 and the summary under the string with D816V. it chose the summary under D816 here

Reply:
same for D816F below